website/src/components/content/WhitePaperPage.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import { RequestDemoAction } from '../../css/components/actions/RequestDemoAction'
+import { COLORS } from '../Jumbotron'
+import { ContentPage } from './ContentPage'
+
+interface Quote {
+    quote: string
+    author: string
+    image?: string
+}
+
+interface Props {
+    title: string
+    quote?: Quote
+    pdf?: string
+    className?: string
+    titleClassName?: string
+    children?: React.ReactNode
+    bgImage?: string
+}
+
+export const WhitePaperJumbotron: React.FunctionComponent<{
+    className?: string
+    color?: keyof typeof COLORS
+    children?: React.ReactNode
+}> = ({ className = '', color = 'dark', children }) => (
+    <div className={`jumbotron rounded-0 ${COLORS[color]} ${className}`}>
+        <div className="container text-center pt-5 pb-3">
+            <span className="white-papers__label d-block mt-1">white paper</span>
+            {children}
+        </div>
+    </div>
+)
+
+export const MediaQuote: React.FunctionComponent<{
+    image?: string
+    quote: string
+    author: string
+}> = ({ image, quote, author }) => (
+    <div className="container pt-6">
+        <div className="case-studies__quote row justify-content-center">
+            {image && (
+                <div className="col-12 col-lg-9">
+                    <img className="rounded-circle img-fluid mx-auto d-block pb-3" src={image} alt={author} />
+                </div>
+            )}
+            <div className="col-12 col-lg-9">
+                <blockquote className="blockquote">
+                    <p className="text-light" dangerouslySetInnerHTML={{ __html: quote }} />
+                    <footer className="blockquote-footer text-light mt-1">{author}</footer>
+                </blockquote>
+            </div>
+        </div>
+    </div>
+)
+
+export const WhitePaperPage: React.FunctionComponent<Props> = ({
+    title,
+    quote,
+    className = 'white-paper',
+    titleClassName = '',
+    pdf,
+    children,
+    bgImage,
+}) => (
+    <div
+        className={`${className}`}
+        // tslint:disable-next-line: jsx-ban-props
+        style={bgImage ? { backgroundImage: `url(${bgImage})`, backgroundRepeat: `no-repeat` } : undefined}
```

Comment:
Is there a reason this can't be moved to a stylesheet?

Reply:
This allows to add a custom individual background that can vary with each white paper. If we move it into a stylesheet, we would need a new class for each white paper. Or do you know a modular way?

Reply:
Ah got it, sounds good.